scene/animation/animation_player.h

```diff
@@ -205,6 +215,16 @@ class AnimationPlayer : public AnimationMixer {
 	double get_current_animation_position() const;
 	double get_current_animation_length() const;
 
+	void set_current_marker_section(const StringName &p_start_marker, const StringName &p_end_marker);
+	void set_current_section(double start_time, double end_time);
```

Comment:
```suggestion
	void set_current_section(double p_start_time, double p_end_time);
```
Prefix arguments with `p_`